VERSION

```diff
@@ -1 +1 @@
-0.0.8
+0.0.9
```

Comment:
Lets use 0.1.1 please.
0.0.8 was a mistake, we should have use 0.1.0 back then instead and now is a good time to set it correctly.

Reply:
You wrote yourself that this is not the place to discuss/implement a new versioning scheme, therefore this is IMHO not a valid objection.

Reply:
It is valid if I consider it wrong.

Reply:
You did not explain at all why following the previously agreed upon procedure is suddenly wrong or why even using `0.0.9` for now is wrong. You only wrote that you would prefer an even bigger version increment which continues to be possible even after merging the current PR as-is. And deciding on a new versioning scheme is as you agreed discussed in #522.

Valid objections iMHO would be if the version would be too high (because we cannot easily go back to lower versions) or if the file would be syntactically broken.

Reply:
I will try to explain why `0.0` versions are no longer valid, although I feel that it is a replay from other discussions.

`0.0.8` was a special version, it got added into RHEL 8.1 and became the first version we need to support.
We did a mistake with its naming, as we should have called it 0.1.0 and open the option to have fixes (or enhances) that are tailored to it. After we release 0.0.8 we continued with bug fixes and enhancements, but we could not backport all of them (rebase) to be supported by 0.0.8, only specific changed got backported. At that point the `stable_0.0.8` was created to manage those backports and to be able and release build from it into RHEL. We should have released intermediate builds upstream so the "community" could test it, but we have not.

Consider these platform distributions our milestones, which we need to support them for some time on upstream (at least until the next version of the platform is released).

If such a "milestone" is marked using a minor version ID (e.g. 0.1), we can fix stuff or add backported enhancement from master and at the same time continue with master for the next milestone (e.g. 0.2).

With my nack I claim that `0.0.9` is not a valid version ID for master, and I think it is valid to ask not to see such a version there.

I understand if you cannot trust only my proposal and will prefer to wait for @cathay4t to finalize this. I am ok with waiting.

Reply:
> With my nack I claim that `0.0.9` is not a valid version ID for master, and I think it is valid to ask not to see such a version there.

After `0.0.8` was released/tagged, it became an invalid version for the master branch, so before any new commits, `VERSION` needed to be bumped. Since it did not happen, dozens of invalid packages were already published as an invalid pre-release for `0.0.8` but they are all post releases, see:

https://copr.fedorainfracloud.org/coprs/nmstate/nmstate-git-fedora/builds/

So, even if master is never released as 0.0.9, it is not worse than the current situation, since master is also not going to be released as 0.0.8 (again).

Also with every other versioning scheme, there is no guarantee that the version in `VERSION` will be the next tagged released since if we would be using semver, it might be that we bump the patch level after a release but need to bump the minor release as well before doing the next release.

The discussion about which version to use belongs in the other PR and I consider it more important to fix the versioning for COPR for now than to discuss the next versioning before doing this.